cmd/iaviewer/main.go

```diff
@@ -101,11 +103,15 @@ func PrintDBStats(db dbm.DB) {
 
 // ReadTree loads an iavl tree from the directory
 // If version is 0, load latest, otherwise, load named version
-func ReadTree(dir string, version int) (*iavl.MutableTree, error) {
+func ReadTree(dir string, version int, prefix []byte) (*iavl.MutableTree, error) {
 	db, err := OpenDB(dir)
 	if err != nil {
 		return nil, err
 	}
+	if prefix != nil {
```

Comment:
```suggestion
	if len(prefix) != 0 {
```

Reply:
this will also catch a case when prefix is an empty string.

Reply:
good case